Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionForcePointFirewall.yaml

```diff
@@ -0,0 +1,473 @@
+            | where isnotempty(Message)
+            | lookup ActionLookup on DeviceAction
+            | lookup ProtocolLookup on Protocol
+            | lookup SubTypeLookup on DeviceEventClassID
+            | lookup EventResultMapping on Message
+            | extend DvcAction = coalesce(DvcAction,DvcAction1, DvcAction2), 
+                     EventResult = coalesce(EventResult, EventResult1, EventResult2), 
+                     EventSeverity = coalesce(EventSeverity, EventSeverity1)
+            | project-away DvcAction1, EventSeverity1, EventResult1, DvcAction2, EventResult2
+            | lookup ApplicationProtocolLookup on ApplicationProtocol
+            | extend 
+                    EventCount = toint(1),
+                    EventType = "NetworkSession",
+                    EventSchema = "NetworkSession",
+                    EventSchemaVersion = "0.2.6"
+            | parse-kv AdditionalExtensions as (
+                        requestURL:string
+                    ) with (pair_delimiter=';', kv_delimiter='=')
+            | project-rename
+                    EventVendor = DeviceVendor,
+                    EventProduct = DeviceProduct,
+                    EventOriginalType = DeviceEventClassID,
+                    DstPortNumber = DestinationPort,
+                    DstIpAddr = DestinationIP,
+                    SrcPortNumber = SourcePort,
+                    SrcIpAddr = SourceIP,
+                    DstNatIpAddr = DestinationTranslatedAddress,
+                    DstNatPortNumber = DestinationTranslatedPort,
+                    SrcNatIpAddr = SourceTranslatedAddress,
+                    SrcNatPortNumber = SourceTranslatedPort,
+                    EventProductVersion = DeviceVersion,
+                    EventMessage = Message,
+                    DvcOriginalAction = DeviceAction,
+                    SrcBytes = SentBytes,
+                    DstBytes = ReceivedBytes,
+                    EventOriginalSubType = DeviceFacility,
+                    DvcId = DeviceExternalID,
+                    DvcInboundInterface = DeviceInboundInterface,
+                    DvcOutboundInterface = DeviceOutboundInterface,
+                    DvcIpAddr = DeviceAddress,
+                    EventOriginalSeverity = LogSeverity,
+                    DvcDomain = Computer        
+            | extend
+                    EventStartTime = todatetime(ReceiptTime),
+                    EventEndTime = todatetime(ReceiptTime),
+                    boolHasIP = iif(indexof_regex(requestURL,@"(\d+.\d+.\d+.\d+)") == -1, false, true),
+                    boolHasPort = iif(requestURL has ':', true, false)
+            | extend 
+                     NetworkRuleName = coalesce(DeviceCustomString1, //RuleId
+                                                DeviceCustomString2), //NatRuleId
+                     ThreatId = coalesce(DeviceCustomString3, //VulnerabilityReferences
```

Comment:
Done.
Added VirusId to AdditionalFields. I haven't seen this value getting populated in most cases. But since it was mentioned in the documentation, I have captured it in the Parser.